### PR TITLE
Updating README-sycl.md to capture the 3.5 modifications

### PR DESCRIPTION
**Pull request by mehdi-goli**

### CODE REVIEW

File: README-sycl.md [hunk 1]
```diff
@@ -44,18 +44,30 @@ make -G Ninja  \
...
 
 You can run it like this from your build directory
 
 ```
-LD_LIBRARY_PATH=/path/to/sycl/install/lib ./examples/cute/tutorial/sgemm_nt_1_sycl
+LD_LIBRARY_PATH=/path/to/sycl/install/lib ./examples/cute/tutorial/[EXAMPLE_NAME]_sycl
 ```
 
+## CUTLASS Example
+ Currently, the the example `14_amper_tf32_tensorop_gemm` has been implemented for SYCL on Nvidia Ampere architecture. You can build this from your build directory by running :
```
Comment (aacostadiaz) [inline README-sycl.md:61]:
```suggestion
 Currently, the example `14_amper_tf32_tensorop_gemm` has been implemented for SYCL on Nvidia Ampere architecture. You can build this from your build directory by running :
```

File: README-sycl.md [hunk 2]
```diff
@@ -17,14 +17,14 @@ resources for GPUs.
...
 
-## Building with SYCL support
-
-To build CUTLASS SYCL support you need the latest version of DPC++ compiler, 
-you can either use a recent [nighly build](https://github.com/intel/llvm/releases)
+## Requirements
+ 
+To build CUTLASS SYCL support you need the latest version of DPC++ compiler, you can either use a recent [nighly build](https://github.com/intel/llvm/releases)
 or build the compiler from source.
 In either case, make sure to enable the NVIDIA plugin so you can build applications
```
Comment (AD2605) [inline README-sycl.md:24]: I do not suppose the Nvidia / AMD plugins can be installed unless the base toolkit is installed.
For this we need to build dpcpp from source with the Cuda backend enabled

File: README-sycl.md [hunk 3]
```diff
@@ -17,14 +17,14 @@ resources for GPUs.
 
 Currently, only one example works on NVIDIA SM 80.
 
-## Building with SYCL support
-
-To build CUTLASS SYCL support you need the latest version of DPC++ compiler, 
-you can either use a recent [nighly build](https://github.com/intel/llvm/releases)
+## Requirements
+ 
+To build CUTLASS SYCL support you need the latest version of DPC++ compiler, you can either use a recent [nighly build](https://github.com/intel/llvm/releases)
```
Comment (AD2605) [inline README-sycl.md:22]:
```suggestion
To build CUTLASS with SYCL support, install the latest DPC++ compiler with the CUDA backend enabled, as described [here](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#build-dpc-toolchain-with-support-for-nvidia-cuda), and make sure to update the following environment path as follows - 

`export PATH=/path/to/llvm/build/install/bin:$PATH`
`export LD_LIBRARY_PATH=/path/to/llvm/build/install/lib:$LD_LIBRARY_PATH`
```

File: README-sycl.md (hunk 2 above)
Comment (mehdi-goli) [reply README-sycl.md:24]: No you dont need to you can the plugins from codpelay nightly build.

Comment (AD2605) [reply README-sycl.md:24]: I see, in that case let's not call it plugins, which confuses with the Plugins available on the codeplay's website to people who are completely new to SYCL,

we can phrase it as - 
```suggestion
To build CUTLASS with SYCL support, install the latest DPC++ compiler with the CUDA backend enabled, either by building from source as described [here](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#build-dpc-toolchain-with-support-for-nvidia-cuda) ,  or by downloading the [nightly releases](https://github.com/intel/llvm/releases)
```

**Review by AD2605 — APPROVED** [review]

**Pull request merged**